k8s/staging/ethereum-deploy.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
It seems like `staging/ethereum-deploy.yaml`, `testnet/ethereum-deploy.yaml`, and `acceptance/ethereum-deploy.yaml` are all the same. Can we put them in `shared/ethereum-deploy.yaml`? It will DRY up some YAML and prepare us for the eventual extraction of ethereum from CL.